images/metering-ansible-operator/roles/meteringconfig/tasks/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
-- name: start metering
-  debug:
-    msg: "Hello World (non default)"
+
+- name: Extract MeteringConfig spec
+  copy: content="{{ _metering_openshift_io_meteringconfig.spec }}" dest=/tmp/metering-values.yaml
+
+- name: Install Metering
+  shell: |
```

Comment:
`command` is typically the preferred module

Reply:
I'm using a pipe here `|`, so I need `shell`

Reply:
And I'm also redirect both stdout and stderr to the pipe (normally it's just stdout) using `2>&1`